gapis/service/service.proto

```diff
@@ -1119,8 +1119,7 @@ message TraceOptions {
   uint32 start_frame = 12;
   // How many frames should we capture
   uint32 frames_to_capture = 13;
-  // Disable disable pre-compiled shaders.
-  bool disable_pcs = 14;
+  reserved 14;
```

Comment:
Remove it in the `DeviceTraceConfiguration` message as well.

Reply:
Woops, sorry for missing this one out, done.